Felienne/hedy
# Implemented user achievements

**Description**
In this PR we implement user achievements. Small goals user can work towards and when reached unlock the achievement and receive a badge. Due to the complex structure of this project it is already stored here as a PR draft to keep track of WiP and possible errors. For a complete overview of the achievements that will be implemented as well as the current state and thoughts on the implementation, notice the discussion [here](https://github.com/Felienne/hedy/discussions/1541).

**Fix for**
This PR fixes #1516.

**How to test**
Notice the new profile option on the 'My-profile' hover that directs to the achievements page. This page contains an overview of the already received (and still to get) achievements.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
